tests/snuba/api/endpoints/test_group_events_oldest.py

```diff
@@ -41,21 +41,18 @@ def setUp(self):
 
         self.group = Group.objects.first()
 
-    def test_simple(self):
+    def test_snuba_no_environment(self):
+        options.set('snuba.events-queries.enabled', True)
```

Comment:
Were these tests even hitting snuba before?

Reply:
`test_snuba_no_environment` was basically doing this before